packages/flutter/lib/src/widgets/page_view.dart

```diff
@@ -442,19 +442,27 @@ class PageView extends StatefulWidget {
   ///
   /// [itemBuilder] will be called only with indices greater than or equal to
   /// zero and less than [itemCount].
+  ///
+  /// Providing a [findItemIndex] lets the underlying [SliverChildBuilderDelegate]
```

Comment:
Actually, lets remove the findItemIndex functionality from PageView.builder. If people want this, they might as well use pageView.custom.